src/translations/ca.translation

```diff
@@ -1,1359 +1,1361 @@
         <name>stgCourseDescriptionCopyEmailSubject</name>
+        <label>Descripció del curs Còpia Treball Complet amb un estatus de:</label>
     </customLabels>
     <customLabels>
-        <label>Domini predeterminat de l&apos;idioma de contacte</label>
         <name>stgDefaultContactLanguageFluency</name>
+        <label>Domini predeterminat de l'idioma de contacte</label>
     </customLabels>
     <customLabels>
-        <label>Tipus predeterminat de registre de la facultat</label>
         <name>stgDefaultFacultyTypeTitle</name>
+        <label>Tipus predeterminat de registre de la facultat</label>
     </customLabels>
     <customLabels>
-        <label>Tipus predeterminat de registre d&apos;estudiant actiu</label>
         <name>stgDefaultStudentTypeTitle</name>
+        <label>Tipus predeterminat de registre d'estudiant actiu</label>
     </customLabels>
     <customLabels>
-        <label>Desactiva el maneig d&apos;errors</label>
         <name>stgDisableErrorHandlingTitle</name>
+        <label>Desactiva el maneig d'errors</label>
     </customLabels>
     <customLabels>
-        <label>Desactiva l&apos;aplicació de correu electrònic preferit</label>
         <name>stgDisablePreferredEmailEnforcement</name>
+        <label>Desactiva l'aplicació de correu electrònic preferit</label>
     </customLabels>
     <customLabels>
-        <label>Desactiva l&apos;aplicació de telèfon preferit</label>
         <name>stgDisablePreferredPhoneEnforcement</name>
+        <label>Desactiva l'aplicació de telèfon preferit</label>
     </customLabels>
     <customLabels>
-        <label><!-- EDA Setup title --></label>
         <name>stgEDASetupTitle</name>
+        <label>Configuració EDA</label>
     </customLabels>
     <customLabels>
-        <label>Activa les connexions del curs</label>
         <name>stgEnableCourseConnectionsTitle</name>
+        <label>Activa les connexions del curs</label>
     </customLabels>
     <customLabels>
-        <label>Activa la funcionalitat de telèfon preferit millorada</label>
         <name>stgEnablePreferredPhoneSync</name>
+        <label>Activa la funcionalitat de telèfon preferit millorada</label>
     </customLabels>
     <customLabels>
-        <label>Notificacions d&apos;errors a:</label>
         <name>stgErrorNotiTo</name>
+        <label>Notificacions d'errors a:</label>
     </customLabels>
     <customLabels>
-        <label>Destinataris notificacions d&apos;errors</label>
         <name>stgErrorNotifRecipientsTitle</name>
+        <label>Destinataris notificacions d'errors</label>
     </customLabels>
     <customLabels>
-        <label>Origen ètnic i racial per als contactes</label>
         <name>stgEthnicityRaceBackfillContacts</name>
+        <label>Origen ètnic i racial per als contactes</label>
     </customLabels>
     <customLabels>
-        <label>Avançat</label>
         <name>stgFluent</name>
+        <label>Avançat</label>
```

Comment:
This field should not be translated. It is a system field used only in the install script and refers to an API name.
```suggestion
        <label>Fluent</label>
```